mp4parse/src/lib.rs

```diff
@@ -3899,19 +3909,27 @@ fn read_dc_descriptor(data: &[u8], esds: &mut ES_Descriptor) -> Result<()> {
     let des = &mut Cursor::new(data);
     let object_profile = des.read_u8()?;
 
+    esds.audio_codec = match object_profile {
+        0x40 | 0x41 => CodecType::AAC,
+        0x69 | 0x6B => CodecType::MP3,
+        _ => CodecType::Unknown,
+    };
```

Comment:
Is it necessary to move this? Doing so potentially changes the behavior even then the `"mp4v"` feature is not enabled since it mutates `esds` before the call to `find_descriptor` which takes it as a parameter

Reply:
Hi @baumanj ,

No, we don't need to move this actually.
I moved this back to the original place to keep the original call flow and committed the change again.

Thanks.